specs/player_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  before do
+    @player = Scrabble::Player.new("kate")
+    @player.play("dog")
+    @player.play("cat")
+    @player.play("rooster")
+  end
+
+  describe '#initialize method' do
+    it "responds to the instance variables " do
+
+      @player.must_respond_to :name
+      @player.name.must_equal "kate"
+    end
+
+    it "Creates a player with an Array of played words" do
+
+      @player.must_respond_to :plays
+      #@player.plays.must_equal ["dog","cat","rooster"]
+      @player.plays.must_be_instance_of Array
+    end
+
+    it "responds to the total_score to 0" do
+
+      @player.must_respond_to :total
+    end
+  end
+
+  describe '#play method' do
+    it "Scrabble::Player responds to plays" do
+
+      @player.must_respond_to :play
+      @player.must_respond_to :plays
+    end
+
+    it "adds an input word to plays array" do
+
+      @player.play("tiger")
+      @player.plays.must_include "tiger"
+    end
+
+    it "returns an array of words" do
+
+      @player.plays.must_equal ["dog","cat","rooster"]
+      @player.plays.must_be_kind_of Array
+    end
+
+
+    it "returns false if the player has won" do
+      @player.play("qqqqqqqq")
```

Comment:
It should only return false if the player has won prior to playing this word.  This is not the case here.